test/plugin/scenarios/webflux-scenario/support-version.list

```diff
@@ -14,21 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-2.0.0.RELEASE
-2.0.1.RELEASE
-2.0.2.RELEASE
-2.0.3.RELEASE
-2.0.4.RELEASE
-2.0.5.RELEASE
-2.0.6.RELEASE
-2.0.7.RELEASE
-2.0.8.RELEASE
-2.0.9.RELEASE
-2.1.0.RELEASE
 2.1.1.RELEASE
```

Comment:
Why remove the versions?

Reply:
because These versions response status sometimes is null.  Test case requires return status code。I think it should be marked as not supported, the support is not very effective

https://github.com/cyejing/skywalking/blob/28fe378fbb5d8781a27a51b42ee77754713deb97/apm-sniffer/optional-plugins/optional-spring-plugins/spring-webflux-5.x-reactive-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/webflux/v5/reactive/DispatcherHandlerHandleMethodInterceptor.java#L93-L100

Reply:
Do you mean response in HTTP level? Not just codes level? I think the cases are stable before you change. So in http network level, it should be stable. Your plugin code just fix the core bug. 

I can't tell the relation between your reference codes and remove supposed version.

Reply:
This code is to get httpstatus.  webflux-2.0.x sometimes get httpstatus is null , 2.1.x all right.  in 2.0.x  really want to get httpstatus, only get netty http status through reflection, which is too complicated. 
Can anyone help？

Reply:
Is this the official confirmed bug? If yes. Then it is not related to our supported version. If it should be different ways, then you should use witness class to use difference APIs for status code.

Which case are you talking about?

Reply:
you are right, Let me check if there is a problem with the source code

Reply:
Let me if you need helps to make decisions. Mostly, we should not reduce the supported versions, our users will be surprised by that.

Reply:
I have found spring-framework issues that fix bugs.
issues: https://github.com/spring-projects/spring-framework/issues/21901
commit: https://github.com/spring-projects/spring-framework/commit/75b1396768079f4044db0fef70dd5f889c1c8931#diff-2beb7fc7f94343be7d8553e833e16113
method: https://github.com/spring-projects/spring-framework/blob/12cb0dfe975e89775e936e06acdfda597a9cab28/spring-web/src/main/java/org/springframework/http/server/reactive/ReactorServerHttpResponse.java#L66-L73

fix in spring-web-v5.1.3.RELEASE  , spring-boot-starter-webflux-v2.1.1.RELEASE

Reply:
Then, we don’t care it, right? We could support status in 2.1.1+ and ignore it in 2.0-2.1.0, right? And the test versions should keep unchanged. Any block for this?

Reply:
Yes, 2.0-2.1.0 Can ignore this state. Then how to write unit tests better, because unit tests cannot be written for different versions. Unless unit tests don't check this status and tags

Reply:
Then, we could remove these versions, but add comments in this version file, describing 2.0-2.1.0 are supported, but due to the status code bug(including the links), we don’t test them. Do you agree this is better?

Reply:
agree, Let me add comments